### PR TITLE
Develop ali

### LINKED ISSUE #4: Add AES Encrypt and Decrypt Methods

### PR DESCRIPTION
**Pull request by alisepehrnia**
Closes #4

### CODE REVIEW

File: SecurityTest/AESTest.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
...am faucibus purus in massa tempor nec feugiat nisl pretium fusce id velit ut tortor pretium viverra suspendisse potenti nullam ac tortor vitae purus faucibus ornare suspendisse sed nisi lacus sed viverra tellus in hac habitasse platea dictumst vestibulum rhoncus est pellentesque elit ullamcorper dignissim cras tincidunt lobortis feugiat vivamus at augue eget arcu dictum varius duis at consectetur lorem donec massa sapien faucibus et molestie ac feugiat sed lectus vestibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt ornare massa eget egestas purus viverra accumsan in nisl nisi scelerisque eu ultrices vitae auctor eu augue ut lectus arcu bibendum at varius vel pharetra vel turpis nunc eget lorem dolor sed viverra ipsum nunc aliquet bibendum enim facilisis gravida neque convallis a cras semper auctor neque vitae tempus quam pellentesque nec nam aliquam sem et tortor consequat id porta nibh venenatis cras sed felis eget velit aliquet sagittis id consectetur purus ut faucibus pulvinar elementum integer enim neque volutpat ac tincidunt vitae semper quis lectus nulla at volutpat diam ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet proin fermentum leo";
```
Comment (siyavashfeyzi) [inline SecurityTest/AESTest.cs:32]: What????

File: Crypto/AES.cs [hunk 2]
```diff
@@ -0,0 +1,133 @@
+﻿using System.IO;
+using System.Security.Cryptography;
+
+namespace Security.Crypto
+{
+    /// <summary>
+    /// Providing methods to encrypt and decrypt data using Advanced Encryption Standard (AES) / Rijndael 
+    /// </summary>
```
Comment (siyavashfeyzi) [inline Crypto/AES.cs:8]: Good.

**Review by siyavashfeyzi — COMMENTED** [review]
The codes are very clean
But SecurityTest class , line 32 was not a concept

File: SecurityTest/AESTest.cs (hunk 1 above)
Comment (alisepehrnia) [reply SecurityTest/AESTest.cs:32]: It's a fake string just to sure about encrypting long strings (search `lorem impsum` for more description)

**Pull request merged**